app/questionnaire/rules/rule_evaluator.py

```diff
@@ -29,7 +29,7 @@ class RuleEvaluator:
     metadata: Optional[MetadataProxy]
     response_metadata: Mapping
     location: Union[None, Location, RelationshipLocation]
-    routing_path_block_ids: Optional[list] = None
+    routing_path_block_ids: Union[Optional[tuple], list[str]] = None
```

Comment:
Always an optional list of strings?. If we don't need it to be list specific, it can be `Iterable/Sequence`

Reply:
Updated now 👍